deploy.sh

```diff
@@ -54,14 +54,19 @@ fi
 SVN_URL="https://plugins.svn.wordpress.org/${SLUG}/"
 SVN_DIR="${HOME}/svn-${SLUG}"
 
-# Checkout just trunk and assets for efficiency
-# Tagging will be handled on the SVN level
+# Checkout SVN repository.
 echo "➤ Checking out .org repository..."
 svn checkout --depth immediates "$SVN_URL" "$SVN_DIR"
 cd "$SVN_DIR"
 svn update --set-depth infinity assets
 svn update --set-depth infinity trunk
+svn update --set-depth immediates tags
 
+# Bail early if the plugin version is already published.
+if [[ -d "tags/$VERSION" ]]; then
+	echo "ℹ︎ Version $VERSION of plugin $SLUG was already published";
+	exit
```

Comment:
@johnbillion Part of the reason we're implementing this is because we _don't_ want the action to fail if the version is already published.

In the `WordPress/performance` repository, we publish several standalone plugins in a single workflow using this action, and we want only those plugins to be published that aren't already published in the given version. So the workflow failing is not right in that scenario, as it's not a failure, but rather a check whether it should be published or not.

Reply:
Thanks @felixarntz for the answer.